src/llvm/lib_llvm.cr

```diff
@@ -31,6 +32,8 @@ end
     IS_38 = {{LibLLVM::VERSION.starts_with?("3.8")}}
 
     IS_LT_70 = IS_38 || IS_39 || IS_40 || IS_50 || IS_60
+    IS_LT_80 = IS_LT_70 || IS_70 || IS_71
+    IS_LT_110 = IS_LT_80 || IS_90 || IS_100 || IS_110
```

Comment:
Is this right?

- `IS_LT_70` does not include `IS_70`
- `IS_LT_80` does not include `IS_80`
- `IS_LT_110` _does_ include `IS_110`

Reply:
🤦  good catch. I think that IS_LT_110 is not needed at the end. I will do another round of manual test for old llvm versions tomorrow and fix this.

Reply:
Done. I now remember that I wanted IS_LT_110 to express accurately when the LLVMBuildCall / LLVMBuildInvoke are not longer available. Since the compiler uses IS_LT_80 to use LLVMBuildCall2 / LLVMBuildInvoke2 the invalid definition of IS_LT_110 went unnoticed.

Thanks